blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -187,6 +187,8 @@ public void testListContainerMarker() throws InterruptedException {
                container);
          assert container.getNextMarker() == null;
 
+         container = view.getBlobStore().list(containerName, afterMarker("z"));
+         assert container.getNextMarker() == null;
```

Comment:
Could you write this as `assertThat(container.getNextMarker()).isNotNull()` for a more informative error message?